File: 389-find-the-difference/389-find-the-difference.js
/**
 * @param {string} s
 * @param {string} t
 * @return {character}
 */
 var findTheDifference = function(s, t) {
   let sum =0;
    for(let i=0;i<t.length;i++){
        sum+= (t.charCodeAt(i)-97);
    }
       for(let i=0;i<s.length;i++){
        sum-= (s.charCodeAt(i)-97);
    }
     
    return String.fromCharCode(sum+97);
     
}

File: 389-find-the-difference/389-find-the-difference.cpp
class Solution {
public:
    char findTheDifference(string s, string t) {
        long long sum =0;
        for(int i=0;i<t.length();i++){
            sum+= (t[i] -'a');
        }
        for(int i=0;i<s.length();i++){
            sum-= (s[i]-'a');
        }
            sum = sum+'a';
        return char(sum);
    }

    
};